extension/src/experiments/index.ts

```diff
@@ -596,7 +575,7 @@ export class Experiments extends BaseRepository<TableData> {
     }
 
     return await pickExperiment(
-      this.experiments.getUniqueList(),
+      this.experiments.getCombinedList(),
```

Comment:
[F] This name had to change as the old name was misleading (data is no longer duplicated)